api/app_analytics/serializers.py

```diff
@@ -16,3 +16,14 @@ class UsageDataQuerySerializer(serializers.Serializer):
 
 class UsageTotalCountSerializer(serializers.Serializer):
     count = serializers.IntegerField()
+
+
+class SDKAnalyticsFlagsSerializerDetail(serializers.Serializer):
+    feature_name = serializers.CharField()
+    identity_identifier = serializers.CharField()
```

Comment:
Maybe this is the default, but this should be `required=False` I think? Flags can be evaluated without the context of an identifier.

Reply:
Nice catch! I've set it to `required=False, default=None` so the client can fail to include the key but it will still save as `None`.